_posts/2018-11-26-resize-text.md

```diff
@@ -27,18 +27,26 @@ When someone resizes text, any of the following can break the page:
 * content is lost or truncated
 * interactive elements overlapping each other
 
+The importance is that *relative units* (_em_, _rem_, _%_, _vw_) are used for text.
```

Comment:
Mind if we use bold instead of italic here to stress the relative unit's importance? I believe a bold text treatment has less an overall chance of being a Dyslexia trigger.

It could also be good to incorporate a link to [relative length units on the MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/length#Units) for those who are unfamiliar.

Reply:
Done!